react-autosuggest/react-autosuggest.d.ts

```diff
@@ -0,0 +1,67 @@
+// Type definitions for react-autosuggest v3.7.3
+// Project: http://react-autosuggest.js.org/
+// Definitions by: Nicolas Schmitt <https://github.com/nicolas-schmitt>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+///<reference path="../react/react.d.ts"/>
+
+declare namespace ReactAutosuggest {
+    import React = __React;
+
+    interface ISuggestionUpdateRequest {
```

Comment:
https://www.typescriptlang.org/docs/handbook/writing-declaration-files.html#naming-conventions

> Naming Conventions
In general, you shouldn’t prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.